native/libcst/src/nodes/statement.rs

```diff
@@ -3490,7 +3497,15 @@ impl<'r, 'a> Inflate<'a> for DeflatedTypeParam<'r, 'a> {
     fn inflate(self, config: &Config<'a>) -> Result<Self::Inflated> {
         let param = self.param.inflate(config)?;
         let comma = self.comma.inflate(config)?;
-        Ok(Self::Inflated { param, comma })
+        let equal = self.equal.inflate(config)?;
+        let default = self.default.inflate(config)?;
```

Comment:
This is not very obvious, but the order of `inflate` calls matter very much because they mutate whitespace that's shared between them. They generally should follow the same order as they are invoked for `codegen`.